### PR TITLE
feat: add landing page

### PR DESCRIPTION
**Pull request by ajay-mandaviya**
create a landing page
Screenshot from 2022-02-12 00-18-35
Screenshot from 2022-02-12 00-18-54
Screenshot from 2022-02-12 00-19-04

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+  padding: 0 2rem;
+  justify-content: space-between;
+}
+
+.search-input {
+  display: flex;
+  align-items: center;
+  border: 2px solid gray;
+  border-radius: 5px;
+  padding: 3px;
```
Comment (ramakrishnach14) [inline style.css:50]: better to use relative units and uniformity in padding like multiple of 4 ,  8

File: style.css [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+  top: 0;
+  right: 0;
+  background: #111827;
+  padding: 0.4rem;
+  color: var(--white);
+  font-size: 0.75rem;
+  font-weight: bold;
+}
+
+@media (max-width: 768px) {
```
Comment (ramakrishnach14) [inline style.css:228]: different screen sizes better to maintain in variables

**Review by ramakrishnach14 — COMMENTED** [review]
looks Good

File: style.css [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+  background-color: var(--white);
+  position: relative;
+}
+.card-img {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.card-subtitle {
+  color: #374151;
```
Comment (shivanigangadharan) [inline style.css:191]: can use css variables for color/bg color.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
```
Comment (shivanigangadharan) [inline index.html:7]: You may also want to add your component library stylesheet so that you can use components from that too.

**Review by shivanigangadharan — APPROVED** [review]

File: style.css (hunk 3 above)
Comment (ajay-mandaviya) [reply style.css:191]: done

File: index.html (hunk 4 above)
Comment (ajay-mandaviya) [reply index.html:7]: use those component but do a to much a customize customized

File: style.css (hunk 1 above)
Comment (ajay-mandaviya) [reply style.css:50]: done

**Pull request merged**